graphql-dgs-reactive/src/main/java/com/netflix/graphql/dgs/reactive/DgsReactiveQueryExecutor.java

```diff
@@ -142,14 +141,38 @@ default <T> Mono<T> executeAndExtractJsonPath(@Language("GraphQL") String query,
      * @param query     Query string
      * @param jsonPath  JsonPath expression.
      * @param variables A Map of variables
+     * @param serverRequest A Spring {@link ServerRequest} giving access to request details.     *
      * @param <T>       The type of primitive or map representation that should be returned.
      * @return A Mono that might contain the resolved type.
      * @see <a href="https://graphql.org/learn/queries/#variables">Query Variables</a>
      * @see <a href="https://github.com/json-path/JsonPath">JsonPath syntax docs</a>
      */
     <T> Mono<T> executeAndExtractJsonPath(@Language("GraphQL") String query,
```

Comment:
Sorry, a little late to review, but have you verified that this api change does not break backwards compatibility?

Reply:
Also, would appreciate any help with documenting this feature, if possible.

Reply:
> Sorry, a little late to review, but have you verified that this api change does not break backwards compatibility?

Great eyes, thank you!
Addressed with https://github.com/Netflix/dgs-framework/pull/1174

Reply:
> Also, would appreciate any help with documenting this feature, if possible.

Glad to!  I'll familiarize myself with the documentation repo and get a PR together.

Reply:
Thanks!

Reply:
https://github.com/Netflix/dgs/pull/106 for my first pass for addition of documentation.